map/games/NWO1915Lebowski.xml

```diff
@@ -3079,7 +3079,7 @@
       <number min="0" max="1000"/>
     </property>
     <!-- Map Name: also used for map utils when asked -->
-    <property name="mapName" value="new_world_order1915lebowski" editable="false"/>
+    <property name="mapName" value="new_world_order_1915lebowski" editable="false"/>
```

Comment:
You've tested this update @CharlieAtlas  and it looks good? I'm a bit surpirsed, I thought the map name needed to match the folder name, which matches the repo name. The repo does not have the underscore before 1915.

Reply:
Yeah, it worked for me.  

I was confused by this whole naming issue.  When i download it, it has the underscore.  When i unzip it, it has a different file structure than the rest of the maps i have though.  It basically had the normal folder structure down a level.  So it was /new_world_order_1915lebowski/new_world_order1915lebowski/maps/games/NWO1915Lebowski.xml.  Notice the missing underscore in the second layer(?) there.  

I initially got it to work by removing the underscore from the folder name (I can't remember if it was the zipped or the unzipped folder).  So if there is a way to do that, it might be the better fix.  But at least for me, this worked, and I could do this.

Reply:
Alright, good work digging into this, I'll trust your findings here. Hopefully one fine day this alignment of folder names and values in XML won't be a thing anymore, until then though...


:+1: